src/mscorlib/shared/System/Numerics/Vector.cs

```diff
@@ -2,9 +2,13 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+#if netcoreapp
+using Internal.Runtime.CompilerServices;
```

Comment:
nit: put this after all the System.* using directives

Reply:
I thought we sort them alphabetically...

Reply:
Yes, but for ones other than System (like internal), we group them separately (with system first). At least that is what I have seen (and followed) elsewhere.

So, something like:
```C#
using System.X;
using System.Y;
using System.Z;

using Internal.A;
using Internal.B;
using Internal.C;
```

Reply:
Between, this PR is already merged and closed. So perhaps we can do all this in the optimization issue along with other clean up?

Reply:
Of course. That would be great.